layers/+completion/cemosv/packages.el

```diff
@@ -0,0 +1,293 @@
+;;; packages.el --- cemosv layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Thanh Vuong <thanhvg@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+(defconst cemosv-packages
+  '(auto-highlight-symbol
+    imenu
+    marginalia
+    (cemosv-spacemacs-help :location local)
+    consult
+    embark
+    embark-consult
+    orderless
+    (selectrum :toggle (eq cemosv-engine 'selectrum))
+    (vertico :toggle (eq cemosv-engine 'vertigo))
```

Comment:
This looks like a typo. Does the value for cemosv-engine need to be `vertico` here instead of `vertigo`. Cause config.el suggests it should be `vertico`